npm/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "opal",
+  "version": "1.1.0",
```

Comment:
That would be another place where the Opal version is defined. Best note that here, for the future releasers: https://github.com/opal/opal/blob/master/lib/opal/version.rb

By the way, 8ef6d934f01a93a70d83559888239d600ab48d87 adds a development version in a root directory. This may need to be reconciled I think.

Reply:
These package.json have different purposes.
One is for installing things to run tests, develop opal.
The other is for building opal packages for npm. (And i will have to update these)